app/controllers/general.php

```diff
@@ -615,10 +615,21 @@ function router(App $utopia, Database $dbForPlatform, callable $getProjectDB, Sw
                 }
             }
 
+            // Truncate logs if they exceed the limit
+            $maxLogLength = APP_FUNCTION_LOG_LENGTH_LIMIT;
```

Comment:
@vermakhushboo Since this is change to Appwrite Router, we need a PR for Edge as well